src/main/resources/db/changelog/db.changelog-master.yml

```diff
@@ -0,0 +1,7 @@
+databaseChangeLog:
+  - include:
+      file: 02-create-roles-table.sql
```

Comment:
Can we start with the first migration `01-create-users-table.sql`?

Reply:
I made the order like that because Users have a Role field, which is created in the 2nd migration. 
I will change the order so Roles are created in the first migration and then the Users table in the second one.